general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -327,13 +372,14 @@ def supercheq(
         return self.post_request("/supercheq", json_dict)
 
     def target_info(self, target: str) -> Dict[str, Any]:
-        """Makes a POST request to the Superstaq API (using the `/target_info` endpoint to request
-        information about `target`.
+        """Makes a POST request to the Superstaq API (using the `/target_info` endpoint to
+        request information about `target`.
```

Comment:
Make 1-liner (unsure how best to split it up)

Reply:
Done in [f98da3c](https://github.com/Infleqtion/client-superstaq/pull/590/commits/f98da3c10266d3819e15851de8e7e4c78304640f)